source/tests/pt_expt/test_multitask.py

```
@@ -0,0 +1,2099 @@
+        )
+        self.assertFalse(
+            finetune_links_false["model_2"].resuming,
+            "model_2 should not be resuming (has finetune_head)",
+        )
+
+        # Phase 4: test change_model_params=True (contrast)
+        ft_config_true = deepcopy(ft_config)
+        # Also set trainable=False to verify it's preserved
+        ft_config_true["model"]["shared_dict"]["my_descriptor"]["trainable"] = False
+        ft_config_true["model"], _ = preprocess_shared_params(ft_config_true["model"])
+        model_config_true, finetune_links_true = get_finetune_rules(
```

Ruff (0.15.10)

[warning] Unpacked variable `finetune_links_true` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)